doc/Caliptra.md

```diff
@@ -894,29 +894,31 @@ Caliptra contains 32 PCR 384 banks that are extendable by the SHA engine, and re
 
 ### Attestation of Caliptra's update journey
 
-The Caliptra hardware ROM will extend Caliptra's FMC measurement and security state into the cumulative and current PCR banks (0 and 1 respectively).  The FMC captures the current measurement of Caliptra's Runtime Firmware upon every cold boot and hitless update. These measurements are used to derive a CDI and an alias key given to runtime firmware. FMC places runtime firmware's measurements into runtime firmware's alias key certificate, and signs that certificate with FMC's alias key.
+The Caliptra hardware ROM extends Caliptra's FMC measurement and security state into the cumulative and current PCR banks (0 and 1 respectively). The FMC captures the current measurement of Caliptra's runtime firmware upon every cold boot and hitless update. These measurements are used to derive a CDI and an alias key given to runtime firmware. FMC places runtime firmware's measurements into runtime firmware's alias key certificate, and signs that certificate with the FMC alias key.
 
-FMC will also extend Runtime Firmware's measurements into PCR0, prior to launching Runtime Firmware.
+FMC also extends the runtime firmware measurements into PCR0 before launching runtime firmware.
 
-When runtime firmware boots following a hitless update, it will have access to the following pieces of data:
+When runtime firmware boots after a hitless update, it has access to the following pieces of data:
 
 * Its CDI, derived by FMC
 * Its alias key, generated by FMC
 * Its alias certificate, signed by FMC
 * PCR\[0\], extended by FMC
```

Comment:
"PCR\[3\], extended by FMC"

Reply:
should be fixed with merge conflict resolution but please open issue if needed.